iml-orm/src/lib.rs

```diff
@@ -75,10 +83,71 @@ pub use tokio_diesel;
 #[cfg(feature = "postgres-interop")]
 pub use r2d2;
 
+#[derive(Debug, Error)]
+pub enum ImlOrmError {
+    #[error(transparent)]
+    R2D2Error(#[from] r2d2::Error),
+    #[error(transparent)]
+    OneshotCanceled(#[from] oneshot::Canceled),
+    #[error(transparent)]
+    AsyncError(#[from] tokio_diesel::AsyncError),
```

Comment:
Does `transparent` mean that there are generated `From` instances included?

Reply:
The from attribute is what generates it.